### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by MufaroMakiwa**

### CODE REVIEW

**Review by michaellawes — APPROVED** [review]

**Review by cenofe — APPROVED** [review]
good job

File: src/main/webapp/components/Auth/Auth.js [hunk 1]
```diff
@@ -29,49 +30,41 @@ const Auth = ({ history }) => {
...
+
+      signInRef.current.classList.add("visible_sign_in");
+      signInRef.current.classList.remove("hidden_sign_in")
+
+      signUpRef.current.classList.remove("visible_sign_up");
+      signUpRef.current.classList.add("hidden_sign_up");
+      ;
+    }
+    console.log("Opened sign in")
+  }
```
Comment (andiov) [inline src/main/webapp/components/Auth/Auth.js:44]: ;

File: src/main/webapp/components/Auth/Auth.js [hunk 2]
```diff
@@ -29,49 +30,41 @@ const Auth = ({ history }) => {
...
+      setDisplaySignUp(false);
+
+      signInRef.current.classList.add("visible_sign_in");
+      signInRef.current.classList.remove("hidden_sign_in")
+
+      signUpRef.current.classList.remove("visible_sign_up");
+      signUpRef.current.classList.add("hidden_sign_up");
+      ;
+    }
+    console.log("Opened sign in")
```
Comment (andiov) [inline src/main/webapp/components/Auth/Auth.js]: Remove logs

File: src/main/webapp/components/Auth/Auth.css [hunk 3]
```diff
@@ -17,9 +31,10 @@
 }
 
 .Auth_title {
-  font-weight: bold;
-  font-size: 2em;
-}
+ 
+
+  }
```
Comment (andiov) [inline src/main/webapp/components/Auth/Auth.css]: Remove the entire selector if there are no properties

File: src/main/webapp/components/Auth/Auth.css [hunk 4]
```diff
@@ -42,12 +57,20 @@
   min-height: 500px;
   height: 100%;
   overflow: hidden;
+
+  padding: 0;
+  margin: 100px;
+  box-sizing: border-box;
+  background-image: linear-gradient(-225deg, #E3FDF5 50%, #FFE6FA 50%);
+
```
Comment (andiov) [inline src/main/webapp/components/Auth/Auth.css]: Nit: remove new line

File: src/main/webapp/components/Auth/Auth.css [hunk 5]
```diff
@@ -57,24 +80,48 @@
...
 }
 
 .Signin_signup {
   width: 100%;
   height: 100%;
   flex-grow: 1;
   position: relative;
-  margin-top: 24px;
+  margin-top: 100px;
+
```
Comment (andiov) [inline src/main/webapp/components/Auth/Auth.css]: Remove

File: src/main/webapp/components/Auth/Auth.css [hunk 6]
```diff
@@ -139,8 +185,11 @@
   padding-top: 8px;
 }
 
-.Sign_in_button_container {
-}
+.Sign_up_button_container 
+  {background: #B8F2E6;}
+
```
Comment (andiov) [inline src/main/webapp/components/Auth/Auth.css]: Format

File: src/main/webapp/components/Projects/Notifications.js [hunk 7]
```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import "./Notifications.css"
+
+
```
Comment (andiov) [inline src/main/webapp/components/Projects/Notifications.js]: Nit; remove one empty line

File: src/main/webapp/components/Projects/Header.js [hunk 8]
```diff
@@ -5,6 +5,8 @@ import AccountCircleIcon from '@material-ui/icons/AccountCircle';
 import NotificationsIcon from '@material-ui/icons/Notifications';
 import Notification from './Notification.js';
 
+
+
```
Comment (andiov) [inline src/main/webapp/components/Projects/Header.js]: Remove

File: src/main/java/com/google/sps/servlets/SignInServlet.java [hunk 9]
```diff
@@ -22,123 +20,143 @@
...
+	private int getLongField(String username, String password, Datastore datastore, String field)
+			throws DatastoreException {
+		// Field will remain empty string if user does not exist
+		int fieldValue = 0;
+
+		// Check if username/password combination is correct
+		Query<Entity> usernameQuery = Query.newEntityQueryBuilder().setKind("User")
+				.setFilter(
+						CompositeFilter.and(PropertyFilter.eq("username", username), PropertyFilter.eq("password", password)))
+				.build();
```
Comment (andiov) [inline src/main/java/com/google/sps/servlets/SignInServlet.java]: Is this correctly formatted?

File: target/classes/META-INF/resources/reducers/projectReducer.js [hunk 10]
```diff
@@ -16,41 +17,55 @@ const initialState = {
...
     case ACTION.REC_TITLE:
       return { ...state, title: action.payload };
+
     case ACTION.CLEAR_REDUCER:
       return { ...state, activeProject: '', onlineProjects: [], title: '',
       offlineProjects: [], activeUsers: [], contacts: [], html: '',
       css: '', js: '', canEdit: false, collaboratorId: '', 
-	      collaboratorName: '' };
+	      collaboratorName: '', collaboratorAvatar: "0"};
+
```
Comment (andiov) [inline target/classes/META-INF/resources/reducers/projectReducer.js]: Format the entire return properly

File: src/main/webapp/components/Auth/Auth.js [hunk 11]
```diff
@@ -447,8 +445,17 @@ const Auth = ({ history }) => {
               {renderSignUp()}
             </div>
           </div>
+
+          <h5><span>New to CollabCode?</span></h5>
+          <button className="Auth_button right_button " onClick={openSignUp}>
+            SIGN UP
+              </button>
```
Comment (andiov) [inline src/main/webapp/components/Auth/Auth.js]: Indent

File: src/main/webapp/components/Auth/Auth.css [hunk 12]
```diff
@@ -186,8 +199,110 @@
...
+  margin-inline-start: 0px;
+  margin-inline-end: 0px;
+  background:#ffe6fa; 
+  padding:0px 10px; 
+
+
+
+
+
+
```
Comment (andiov) [inline src/main/webapp/components/Auth/Auth.css]: Remove empty lines in this entire file for each property.

File: src/main/webapp/components/Editor/Profile.js [hunk 13]
```diff
@@ -1,14 +1,24 @@
 import React, { useEffect, useRef, useState } from 'react';
 import { useDispatch, useSelector } from 'react-redux';
 import ExitToAppIcon from '@material-ui/icons/ExitToApp';
+import AccountCircleIcon from '@material-ui/icons/AccountCircle';
+
```
Comment (andiov) [inline src/main/webapp/components/Editor/Profile.js:5]: nit: remove

File: target/classes/META-INF/resources/components/Projects/Projects.js [hunk 14]
```diff
@@ -258,8 +257,21 @@ const Projects = ({ history }) => {
     return projects;
   }
 
+  // accept when a user sends collaboration request
+  const acceptCollaboration = (notification) => {
```
Comment (andiov) [inline target/classes/META-INF/resources/components/Projects/Projects.js]: Is this needed if it's empty?

**Pull request merged**